python/pyspark/ml/tests.py

```diff
@@ -655,6 +656,20 @@ def test_nested_pipeline_persistence(self):
             except OSError:
                 pass
 
+    def test_write_property(self):
+        lr = LinearRegression(maxIter=1)
+        path = tempfile.mkdtemp()
+        lr_path = path + "/lr"
+        lr.write.save(lr_path)
```

Comment:
I remembered that my initial change didn't change JavaMLWritable and I created a MLWriter instance and tested whether write is a property, it passed. So I made changes to create a more complete test case per discussion with Yanbo.